src/Nixfmt/Util.hs

```diff
@@ -78,3 +79,6 @@ dropCommonIndentation unstrippedLines =
     in case commonIndentation (filter (/=empty) strippedLines) of
             Nothing          -> map (const empty) strippedLines
             Just indentation -> map (fromMaybe empty . stripPrefix indentation) strippedLines
+
+isSpaces :: Text -> Bool
+isSpaces = Text.all (==' ')
```

Comment:
What if there are tabs? https://github.com/NixOS/nix/pull/2911

I suggest replacing it with `Text.all Data.Char.isSpace` (it even is already imported!).
```suggestion
isSpaces = Text.all isSpace
```

Reply:
Nix handles lines consisting entirely of spaces different from lines that contain other kinds of whitespace. This function is intended to match that behavior but it could probably be better documented. I got stuck on finding good names for lines without any characters vs lines consisting of only but at least one space vs lines that could be either.